mg400_bringup/launch/display.launch.py

```diff
@@ -13,47 +13,40 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from pathlib import Path
-from typing import Dict
+# from pathlib import Path
```

Comment:
Is there any reason that this line is keeped as commments?

Reply:
These lines are remove.